app/models/user.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  has_many :posts, dependent: :destroy
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i.freeze
```

Comment:
- [ ] **[OPTIONAL]** devise already takes care of user attributes (email, password), if you checked `config/initializers/devise.rb` file, you will find all email and password configuration and you can edit it there.